Embedding.py

```diff
@@ -0,0 +1,34 @@
+import discord
+from settings import *
+from checks import *
+
+class Embedding(commands.Cog):
+	"""A suite of command providing users with embeds manipulation tools."""
+	def __init__(self, bot):
+		self.bot = bot
+
+	@commands.command()
+	async def embed(self, ctx, *args):
+		"""allows you to post a message as an embed. Your msg will be reposted by the bot as an embed !"""
+		if ctx.channel.name in POLL_ALLOWED_CHANNELS:
```

Comment:
Maybe we should use `channel.id` instead of `channel.name`

Reply:
See #4